{{cookiecutter.project_shortname}}/code/{{cookiecutter.project_package}}/views.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2021 CERN.
```

Comment:
```suggestion
# Copyright (C) 2022 CERN.
```

Reply:
The copyright should be auto-generated like other places. CERN should not claim copyright :-)